src/index.ts

```diff
@@ -0,0 +1,37 @@
+import http = require("http");
```

Comment:
Use import * as 'http' from 'http' instedof required
https://stackoverflow.com/questions/35728117/difference-between-import-http-requirehttp-and-import-as-http-from-htt

Reply:
changed